src/rust/graph-query/src/var_allocator.rs

```diff
@@ -1,5 +1,7 @@
 use fnv::FnvHashMap as HashMap;
 
+// VarAllocator allocates graphql variables, one for each value to be interpolated
+// The values will be sequentially defined from $a to $z, wrapping to $za
```

Comment:
Is the next one going to be `$zb`? Is this a standard thing for GraphQL? Normally when I see this kind of wrapping, it's `x`, `y`, `z`, `aa`, `ab`, `ac`,... `az`, `ba`, `bb`, etc.

Reply:
The next one would be `$zb` that's correct. I don't know that it's standard, it just seems straightforward and efficient to implement.